lib/perl/Genome/Model/Command/Define.pm

```diff
@@ -24,6 +24,9 @@ sub _build_sub_command {
 
     my $model_subclass_name = $class_name;
     $model_subclass_name =~ s/::Command::Define::/::/;
+    my $do_not_create_define_command;
+    Try::Tiny::try { $do_not_create_define_command = $model_subclass_name->do_not_create_define_command; };
```

Comment:
Should this module `use` or `require` `Try::Tiny` before calling `Try::Tiny::try`?